scripts/schedule-barman-montagu-nightly

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+PATH_BB8=$(command -v docker)
```

Comment:
Typo here, I think? Shouldn't that be `command -v bb8`?

Reply:
good catch (oops)